requirements.txt

```diff
@@ -11,7 +11,8 @@ scipy>=1.5.4
 sentencepiece
 tensorboard
 tokenizers==0.10.1
+datasets==2.0.1
```

Comment:
Collecting tokenizers==0.10.1
  Downloading tokenizers-0.10.1-cp38-cp38-manylinux2010_x86_64.whl (3.2 MB)
ERROR: Could not find a version that satisfies the requirement datasets==2.0.1 (from -r requirements.txt (line 14)) (from versions: 0.0.9, 1.0.0, 1.0.1, 1.0.2, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.2.0, 1.2.1, 1.3.0, 1.4.0, 1.4.1, 1.5.0, 1.6.0, 1.6.1, 1.6.2, 1.7.0, 1.8.0, 1.9.0, 1.10.0, 1.10.1, 1.10.2, 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 1.13.2, 1.13.3, 1.14.0, 1.15.0, 1.15.1, 1.16.0, 1.16.1, 1.17.0, 1.18.0, 1.18.1, 1.18.2, 1.18.3, 1.18.4, 2.0.0, 2.1.0)
ERROR: No matching distribution found for datasets==2.0.1 (from -r requirements.txt (line 14))